zos_subsystems/ims/ims_provisioning_wazi/roles/ims_operations_manager/tasks/main.yml

```diff
@@ -0,0 +1,120 @@
+---
+ 
+# Tasks for Operations Manager (OM)
+- name: Operations Manager
+  collections:
+    - ibm.ibm_zos_core
+    - ibm.ibm_zos_ims
+
+  block:
+    - set_fact:
+        bpe_config: '{{ role_path }}/templates/bpe_config.txt'
+        oi_config: '{{ role_path }}/templates/oi_config.txt'
+        om_config: '{{ role_path }}/templates/om_config.txt'
+        rom_config: '{{ role_path }}/templates/rom_config.txt'
+
+    - name: check om value
+      debug:
+        var: om
+
+    # AL: This code works, but OM doesn't start
```

Comment:
Does this comment from AL need to be addressed?

Reply:
I removed this comment to avoid any confusion. The code works and OM does start ok.